source/material_model/interface.cc

```diff
@@ -288,7 +290,6 @@ namespace aspect
         }
     }
 
-
     template <int dim>
     MaterialModelInputs<dim>::MaterialModelInputs(const FEValuesBase<dim,dim> &fe_values,
                                                   const typename DoFHandler<dim>::active_cell_iterator *cell_x,
```

Comment:
I opened an issue for now (#1907) so that we don't forget about it, and I can do it after this pull request is merged.